### DISCUSSION TITLE
Data wrangling in R

### DISCUSSION THREAD

**Discussion by reniunisa** [teamalgoritma/community#440] [Q&A] [answered]
Supposed that we have hundreds of thousands of lines.
And there is a column of categorical data which consists of thousands of different categorical data, let's say flight numbers.
should we convert this into numeric as well, using mutate function?

thanks

**Comment by muhrisman** [comment, marked as answer]
Categorical data does not mirror the "real value" of it. For example if we have categorical data which consists of numbers, it does not mean that the number is the "value" of it. If the case is flight number, I highly suggest to not to convert it to numeric since it can produce misleading information. If flight number is converted to numeric, it means we can calculate the mean, median etc while flight number itself is unique. Unique identity means we cannot involve mathematical proses directly to that entity.

Best Regards,
M. Risman

Hope that helps :)

**Reply by reniunisa** [reply]
thanks for your answer :-)

__Reni
